C-Plus-Plus/README.md

```diff
@@ -87,12 +87,7 @@
 - [Aggressive cows](cp/aggressive_cows.cpp)
 - [Minimum operations to make sum of each column & row of square matrix equal](cp/matrix_min_operations_for_equal_sum.cpp)
 - [Expressing n as a product of prime numbers](cp/prime_factors.cpp)
-- [Maximum of Minimum for every window size](cp/max_of_min_subarray.cpp)
-- [Maximum Sum Subarray of Size k](cp/max_sum_subarray_of_size_k.cpp)
-- [Maximum of all subarray of size k](cp/maximum_of_all_subarray_of_size_k.cpp)
-- [First Negative Integer in Every k sized window](cp/1st_negative_no_in_every_window_of_size_k.cpp)
-- [Rotating an NxN 2D array in 90 degrees anticlockwise direction](cp/rotate2darray.cpp)
-- [Count occurence of Anagram](cp/count_occurences_of_anagrams.cpp)
```

Comment:
Why so many removals?

Reply:
i was facing merge conflict in my main branch beacause of the pr you merged just now so I deleted them from readme.Should I add them again?

Reply:
I have changed it.Have a look!